gossip-base/src/main/java/org/apache/gossip/event/GossipListener.java

```diff
@@ -21,4 +21,7 @@
 
 public interface GossipListener {
   void gossipEvent(Member member, GossipState state);
+
+  /* this listener's name */
+  String name();
```

Comment:
Ideally I would not want to have two methods because this prohibits us from doing lamda's right?